src/defs/accessTokenRepresentation.ts

```diff
@@ -0,0 +1,60 @@
+import AddressClaimSet from './addressClaimSet';
+import {Category} from './resourceServerRepresentation';
+
+export default interface AccessTokenRepresentation {
+  acr?: string;
+  address?: AddressClaimSet;
+  'allowed-origins'?: string[];
+  at_hash?: string;
+  auth_time?: number;
+  authorization?: {
```

Comment:
Can we split these nested types out into separate interfaces?

Reply:
Absolutely! Wasn't sure what was best practice here for the ones that would have only one or two properties.